src/proxy-animation.js

```diff
@@ -534,6 +751,14 @@ export class ProxyAnimation {
       // tracked.
       pendingPlaybackRate: null,
       pendingTask: null,
+      // Record the specified timing since it may be different than the timing
+      // actually used for the animation. When fetching the timing, this value
+      // will be returned, however, the native animation will use normalized
+      // values.
+      specifiedTiming: null,
+      // The normalized timing has the corrected timing with the intrinsic
+      // iteration duration resolved.
+      normalizedTiming: null,
       proxy: this
```

Comment:
Can we add effect to this list since we're using that to cache the proxy effect?

Reply:
done